git/go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/fluxcd/pkg/git
+
+go 1.17
+
+require (
+	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/ProtonMail/go-crypto v0.0.0-20220517143526-88bb52951d5b
+	github.com/fluxcd/gitkit v0.5.1
+	github.com/fluxcd/pkg/gittestserver v0.5.4
```

Comment:
All `fluxcd/pkg` packages must be referenced on disk, see runtime for an example.